### PR TITLE
[AVW-2908] Transaction Review Header

### PR DESCRIPTION
**Pull request by kugel3**
Jira ticket: [ABW-2908](https://radixdlt.atlassian.net/browse/ABW-2908)

## Description
Show the title in the navigation bar in Transaction Review, when scrolling away the other title.

### Notes
We could do the same thing in other places too, like the home screen, it's fairly painless.

## How to test

- Go to transaction view and scroll down far enough for the main title to disappear.
 
## Video

https://github.com/radixdlt/babylon-wallet-ios/assets/123396602/0bad2d48-0b58-46c7-8386-ddc78e864e75

## PR submission checklist
- [x] I have tested account to account transfer flow and have confirmed that it works

[ABW-2908]: https://radixdlt.atlassian.net/browse/ABW-2908?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by GhenadieVP** [comment]
How does it look for incoming transactions from a dApp? it should be like [this](https://app.zeplin.io/project/639c7e7ab391bb15ac5d25fa/screen/65ba7cc9af0242bd0a36bd72)

**Comment by kugel3** [comment]
Well it doesn't look like that, I am just using the navigationTitle property. Will try to use a toolbar item instead

**Comment by kugel3** [comment]
Now it looks like that...

File: RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
-  "originHash" : "3313f884630244f405efa4cdbbcb2486d097b16289931114ea3c6826384f24b9",
```
Comment (GhenadieVP) [inline RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:2]: should this be removed?

File: RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -15,8 +14,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/AppsFlyerSDK/AppsFlyerFramework-Strict",
       "state" : {
-        "revision" : "a22ea3af34d4631123bd54f55f95f7b7a9cd905f",
-        "version" : "6.13.2"
+        "revision" : "f9e6a597252ab719805033a7912c64d2dd1f077e",
```
Comment (GhenadieVP) [inline RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:17]: intentional?

**Review by GhenadieVP — APPROVED** [review]
Niice, just check if the changes in Package.resolved are intentional.

**Pull request merged**